internal/manifest/compose/convert.go

```diff
@@ -5,7 +5,6 @@ import (
 	"io"
 
 	"github.com/deref/exo/internal/manifest"
-	"github.com/deref/exo/internal/providers/docker/compose"
```

Comment:
This PR primarily exists to break this dependency. Importing needs to preserve interpolation expressions and doesn't need to deeply understand the configuration within services or other components.